### ISSUE TITLE
Small UX issue: clicking the "input" area in the REPL doesn't always bring focus to the text input

### ISSUE THREAD

**Issue by pauloliver** [ramda/ramda.github.io#96]
If the Input text area loses focus, clicking anywhere in the input column doesn't bring focus back to the text input; you must instead click inside where a current line of code is.  

Below is the `div` that I think is capturing the click.
Current:
![current](https://cloud.githubusercontent.com/assets/1124272/17068914/8a4fd0f6-5008-11e6-90d1-486546908e18.png)

This used to work before #83, you can view the old behavior by checking out commit `fb0cbc0`

Old:
![old](https://cloud.githubusercontent.com/assets/1124272/17068918/9186df68-5008-11e6-9637-79a0c3d46e93.png)

I'm assuming this is from changing 
```CSS
.CodeMirror {
    height: 500px;
}
```
to 
```CSS
.CodeMirror {
    height: 100%;
}
```
in https://github.com/ramda/ramda.github.io/blob/master/repl/lib/css/terminal.css

I'm happy to poke at this a bit later and submit a PR, but I am by no means a CSS expert, so no promises :laughing:

**Comment by buzzdecafe** [comment]
nice find -- i have also found this behavior irritating, so i would welcome a PR if you're up to it. setting the height of a div to 100% is one of things that should be easy, but isn't.

**Comment by CrossEye** [comment]
Yes, this would be a nice win if someone can find a good solution.

**Comment by buzzdecafe** [comment]
closed by #112
